runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDo.java

```diff
@@ -391,19 +392,19 @@ public PipelineOptions getPipelineOptions() {
       };
     }
 
-    /** Creates an {@link DoFn.ExtraContextFactory} that provides just the given tracker. */
-    private DoFn.ExtraContextFactory<InputT, OutputT> wrapTracker(
+    /** Creates an {@link Arguments} that provides just the given tracker. */
+    private Arguments<InputT, OutputT> wrapTracker(
         TrackerT tracker, DoFn<InputT, OutputT>.ProcessContext processContext) {
-      return new ExtraContextFactoryForTracker<>(tracker, processContext);
+      return new ArgumentsForTracker<>(tracker, processContext);
     }
 
-    private static class ExtraContextFactoryForTracker<
+    private static class ArgumentsForTracker<
```

Comment:
Don't have a strong feeling about it; this PR is just a rename. Definitely seems possible to revisit.